src/TosCheck.jsx

```diff
@@ -145,7 +145,7 @@ const showTos =
   (!agreementsForUser.length ||
     agreementsForUser[agreementsForUser.length - 1].value < latestTosVersion);
 
-if (agreementsForUser) {
+if (agreementsForUser && recordToC) {
```

Comment:
We'll still need `recordToC` method. If you don't like this solution I'll remove it totally from here @charleslavon